libs/ledger-live-common/src/exchange/swap/hooks/useSwapTransaction.ts

```diff
@@ -161,6 +162,8 @@ export const useSwapTransaction = ({
     setExchangeRate,
   });
 
+  const ptxSwapMoonpayProviderFlag = useFeature("ptxSwapMoonpayProvider");
+
```

Comment:
I am not sure about have this info in the hook, we are trying to simplify this logic.  Could we add this FF directly in ledger-live-desktop? what do you think?